docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst

```diff
@@ -0,0 +1,369 @@
+
+Set up the docker networks
+--------------------------
+
+Set the specific interface pools by running the following commands.
+
+.. code-block:: bash
+
+    docker network create --subnet=10.1.0.0/16 talker_net
+    docker network create --subnet=10.2.0.0/16 listener_net
+    docker network create --subnet=10.3.0.0/16 wan_net
+
+The three networks would have been created, all of them isolated from the others.
+In order to enable the communication between networks, it is mandatory to update the system ``iptables``.
+
+.. code-block:: bash
+
+    sudo iptables -I DOCKER-ISOLATION-STAGE-2 -i talker_net -o listener_net -j ACCEPT
+    sudo iptables -I DOCKER-ISOLATION-STAGE-2 -o talker_net -i listener_net -j ACCEPT
+    sudo iptables -I DOCKER-ISOLATION-STAGE-2 -i talker_net -o wan_net -j ACCEPT
+    sudo iptables -I DOCKER-ISOLATION-STAGE-2 -o talker_net -i wan_net -j ACCEPT
+    sudo iptables -I DOCKER-ISOLATION-STAGE-2 -i listener_net -o wan_net -j ACCEPT
+    sudo iptables -I DOCKER-ISOLATION-STAGE-2 -o listener_net -i wan_net -j ACCEPT
+
+This set of commands is enabling both *input* (``-i``) and *output* (``-o``) tables to allow both sides communication between the three networks.
+
+XML configuration files
+-----------------------
+
+It is mandatory to set the discovery *server* and the *client* nodes with the following configuration options in order to enable the TCP communication, regardless of whether the deployment is over Docker networks or WAN.
+
+Both XML configuration files will be later used in the :ref:`docker compose <tutorials_deployment_ds_wan_tcp_docker_compose>` instructions to perform the containers deployment.
+
+Server side
+^^^^^^^^^^^
+
+The following XML configuration describes the discovery *server* TCP configuration required.
+Create a workspace to run the tutorial, and include an XML configuration file named ``server_configuration.xml`` with the below code.
+
+.. literalinclude:: /resources/tutorials/core/deployment/ds_wan_tcp_tutorial/server_configuration.xml
+    :language: XML
+
+Note that in the discovery configuration section, the profile is described as *server*, with a specific GUID prefix, and listening locator.
+This listening locator is configured with the IP ``10.1.1.1``, which belongs to the ``talker`` node LAN, and its physical port is included in the transport descriptor as a known listening port.
+This, along setting the wan address with the previously mentioned IP address, and setting the transport descriptor and locator types as ``TCPv4``, is crucial to ensure the TCP communication.
+
+Client side
+^^^^^^^^^^^
+
+Include the following XML configuration in the workspace, and name the file as ``talker_configuration.xml``.
+This former configuration describes the ``talker`` node configuration for the EDP phase, and transport layer.
```

Comment:
Not only EDP, for instance, the discovery server list is mostly used during `PDP`. We could refer as discovery phase in general